.vscode/alleluia.code-snippets

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
maybe you can do snippets like this in your editor, it speeds everything up **a lot**